jsonapi/marshal.go

```diff
@@ -204,6 +211,16 @@ func marshalData(element MarshalIdentifier, data *Data, information ServerInform
 	data.ID = element.GetID()
 	data.Type = getStructType(element)
 
+	if information != nil {
+		if customLinks, ok := element.(MarshalCustomLinks); ok {
+			data.Links = make(map[string]interface{})
+			base := getLinkBaseURL(element, information)
+			for k, v := range customLinks.GetCustomLinks(base) {
+				data.Links[k] = v
```

Comment:
i think custom links should not be allowed to overwrite automatically generated links - what do you think? At least we should add a test either way to document behaviour for this case...